R/draws.R

```diff
@@ -1,10 +1,112 @@
-#' TODO
+#' @title Fit the base imputation model and get parameter estimates
+#'
+#' @description Assuming MAR, `draws` fits the base imputation model to the observed data
```

Comment:
I would start with the assumption perhaps:

> Fits the base imputation model to the observed data only using the observations that can be assumed to be MAR

or something like that ?

Reply:
Fits the base imputation model to the observed data assuming MAR? I just want to specify what we mean by "assuming MAR" in details later.

Reply:
Is "assumed" right, its more that it only uses MAR data rather than makes the assumption ?

Reply:
I guess we are assuming MAR, meaning that we fit the model assuming observed data are MAR.

From Cro et Al:
"..assuming MAR, fit a MVN distribution with an unstructured mean (ie, a separate mean for each of the baseline and post-randomization observation times) and variance-covariance matrix using a Bayesian approach.."

Reply:
fair enough :)